packages/react-reconciler/src/ReactFiberDevToolsHook.js

```diff
@@ -54,6 +56,25 @@ export function injectInternals(internals: Object): boolean {
   try {
     const rendererID = hook.inject(internals);
     // We have successfully injected, so now it is safe to set up hooks.
+    if (__DEV__) {
+      // Only used by Fast Refresh
+      if (typeof hook.onScheduleFiberRoot === 'function') {
+        onScheduleFiberRoot = (root, children) => {
+          try {
+            hook.onScheduleFiberRoot(rendererID, root, children);
+          } catch (err) {
+            if (__DEV__ && !hasLoggedError) {
+              hasLoggedError = true;
+              warningWithoutStack(
+                false,
+                'React DevTools encountered an error: %s',
```

Comment:
This warning is a bit misleading.

Reply:
It’s just copy paste from other “DevTools hooks”. We can change the message in all of them? I don’t think this one is particularly special other than the fact that it happens to be used by Fresh today. 

Technically there’s nothing preventing DevTools from using this one too — in fact I think it might help us remove some fragile logic in the renderer. (For new versions at least.) In that case we’d need to remove the DEV-only flag check though. 

In either case the issue would be filed in the same repo.

Reply:
In other words, we could apply the same argument to the “commit root” hook which has the same message but is used by both DevTools and Fresh.

Reply:
I know it was copy pasted from the previous one, but in this particular case it's *always* incorrect.

Reply:
I don't understand so much pushback against a wording suggestion :smile:

Reply:
I don't think it matters but I'm happy to stamp a PR that changes it :-) I don't mean to "push back", I was only explaining my reasoning for *why* I left it as is. I'm sorry if that explanation isn't useful.

Reply:
It's cool. Thanks for the follow up PR.